addons/pylons/functions/fnc_configurePylons.sqf

```diff
@@ -69,6 +69,6 @@ if (_currentPylon == count _pylonsToConfigure) exitWith {};
         [format [localize LSTRING(Stopped), _currentPylon + 1], false, 5] call EFUNC(common,displayText);
     },
     format [localize LSTRING(ReplacingPylon), _currentPylon + 1, count _pylonsToConfigure],
-    {true},
+    {(ace_player distanceSqr GVAR(currentAircraft)) <= GVAR(searchDistanceSqr)},
```

Comment:
This can be opened via a zeus module so it probably needs to be something like
`(!isNull curatorCamera) || (ace_player distanceSqr GVAR(currentAircraft)) <= GVAR(searchDistanceSqr)}`

Reply:
Could just use `GVAR(isCurator)` like line 25 does, the function already relies on it.